lib/formats/odata.js

```diff
@@ -103,6 +103,12 @@ const template = (body) => {
 ////////////////////////////////////////////////////////////////////////////////
 // MESSAGES
 
+const odataXmlError = template(`<?xml version="1.0" encoding="UTF-8"?>
+<error code="{{problemCode}}">
+  <message>{{message}}</message>
+  <details>{{problemDetails}}</details>
```

Comment:
Will `{{problemDetails}}` become `[object Object]`, or is it automatically JSON-stringified?

Reply:
uh yeah they'll probably end up object object. maybe i'll just take the details out for now, i don't know of any errors this could trip that would have meaningful details.